index.html

```diff
@@ -3204,14 +3034,14 @@ <h2>Compression method 0</h2>
 bit in the final block identifies it as the last block, allowing
 the decoder to recognize the end of the compressed datastream.
 Further details on the compression algorithm and the encoding are
-given in the deflate specification [[rfc1951]].</p>
+given in the <a>deflate</a> specification [[rfc1951]].</p>
 
 <p>The check value stored at the end of the zlib datastream is
 calculated on the uncompressed data represented by the
 datastream. The algorithm used to calculate this is not the same
 as the CRC calculation used for PNG chunk CRC field values. The
 zlib check value is useful mainly as a cross-check that the
-deflate and inflate algorithms are implemented correctly.
+<a>deflate</a> algorithms are implemented correctly.
```

Comment:
Here, too.

Reply:
Should now be "deflate algorithm is implemented correctly"